types/spotify-web-playback-sdk/index.d.ts

```diff
@@ -137,6 +137,7 @@ declare namespace Spotify {
         name: string;
         getOAuthToken(cb: (token: string) => void): void;
         volume?: number | undefined;
+        enableMediaSession?: boolean | undefined;
```

Comment:
having `?` and `| undefined` feels redundant?

Reply:
I see what you mean, but I thought it would be better to cover more patterns since they don't behave exactly the same and DefinitelyTyped is used by many people.